ci/taos/plugins-base/pr-audit-build-android.sh

```diff
@@ -57,7 +57,23 @@
 # vi ~/.bashrc
 # # gst-android prebuilt binary (e.g., .a, .so, .h)
 # export GSTREAMER_ROOT_ANDROID=/var/www/html/android/gst_root_android/
+#----------------------------------------------------------------------
```

Comment:
@jaeyun-jung , Please keep the writing style of the existing manual by checking the "Prerequisites" statement.
For example, 

* Before : Step 1/2 , 2/2
* After   : Step 1/4 , 2/4 , 3/4, 4/4

Reply:
changed to 'Step #/4:'